--- a/debian/control	2024-01-30 19:15:52.177436184 +0000
+++ b/debian/control	2024-01-30 19:16:23.005940072 +0000
@@ -14,7 +14,7 @@
                libkrb5-dev | heimdal-dev,
                libpam0g-dev | libpam-dev,
                libselinux1-dev [linux-any],
-               libssl-dev (>= 1.1.1),
+               libssl-dev (>= 3.0.0),
                libsystemd-dev [linux-any] | libelogind-dev [linux-any],
                libwrap0-dev | libwrap-dev,
                pkg-config,
